src/dashboard/cd-dashboard-news.vue

```diff
@@ -68,12 +68,12 @@
     },
     methods: {
       async loadNews() {
-        const res = await NewsForumsService.loadNews({ per_page: 6 });
+        const res = await UpdatesService.loadNews({ per_page: 6 });
```

Comment:
You can avoid loadedPosts by differenciating the value of news from null to [] when there is no result.